ast.go

```diff
@@ -158,18 +239,30 @@ type FunctionAnnotation struct {
 	Options  []Option // Optional: LiteralOption or VariableOption
 }
 
+func (fa FunctionAnnotation) String() string {
+	if len(fa.Options) == 0 {
+		return fmt.Sprint(fa.Function)
+	}
+
+	return fmt.Sprintf("%s %s", fa.Function, sliceToString(fa.Options, " "))
+}
+
 type PrivateUseAnnotation struct {
 	Annotation
 
 	// TODO: Implementation
 }
 
+func (PrivateUseAnnotation) String() string { return "TODO" }
```

Comment:
This breaks MF2 syntax. Use something like `.|private annotation not implemented|`.